resources/views/create.blade.php

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
```

Comment:
Extend the `admin.blade` layout, don't do a brand new html. Check [this](https://laravel.com/docs/5.7/blade#extending-a-layout).